server/util/aria.test.js

```diff
@@ -0,0 +1,1914 @@
+const { hashTest, hashTests } = require('./aria');
+
+describe('Verify test hashes are matching as expected', () => {
```

Comment:
I've added some additional comments in https://github.com/w3c/aria-at-app/pull/621/commits/1bdf4ecd24eb51dc383eea52c1ac3a576c901bc4 to `aria.test.js`, which should make it a bit clearer about what the JSON data being tested is representing (which is test data changes found in the [tests/alerts directory](https://github.com/w3c/aria-at/commits/master/tests/alert)).
Would that address this concern?

Reply:
I don't think so, but maybe I'm misunderstanding.

This project uses hashed representations of data structures as an identity for those structures. Because there is no standard approach to generating such hashes, the project currently has an implicit dependency on the proprietary behavior of the `object-hash` npm module. The implicit nature of this dependency make it difficult for others to recognize because changing to a different hashing technique (e.g. from a new release of `object-hash`, a different library, or even new hashing logic written specifically for this project) might be technically functional, but it would cause a lot of inefficient behavior.

I'm suggesting that we add a unit test which asserts the hashed representation of some arbitrary value, e.g.

```js
it('should produce hashed representations which match those used in production', () => {
  expect(hashTest(singleTest)).toEqual('c0ffee');
});
```

A test like that would fail if the hashed representations change for any reason (including those suggested above), alerting contributors about the inefficient behavior that the change would cause if deployed to production.

Reply:
@jugglinmike Got it! Thanks for the explanation.

> A test like that would fail if the hashed representations change for any reason (including those suggested above), alerting contributors about the inefficient behavior that the change would cause if deployed to production.

https://github.com/w3c/aria-at-app/pull/621/commits/c18afa8599d101a8253ddd21bf33f96d683ff6e1 should address this.

Reply:
Indeed it does!